### PR TITLE
added stopPXValidateDeleteApps test

### PR DESCRIPTION
**Pull request by spai-px**
What this PR does / why we need it:
Adds stopPXValidateDeleteApps

Which issue(s) this PR fixes
Closes # PTX-20645

**Special notes for your reviewer**:
[Aetos dashboard](https://aetos.pwx.purestorage.com/resultSet/testSetID/417250)

### CODE REVIEW

File: tests/basic/pure_test.go [hunk 1]
```diff
@@ -1014,3 +1014,78 @@ var _ = Describe("{RestartPXWhileVolCreate}", func() {
...
+			log.FailOnError(err, "error removing label on node [%s]", selectedNode.Name)
+		}()
+
+		Inst().AppList = []string{"nginx-fada-repl-vps"}
+		err = Inst().S.AddLabelOnNode(selectedNode, k8s.NodeType, k8s.ReplVPS)
+		log.FailOnError(err, fmt.Sprintf("Failed add label on node %s", selectedNode.Name))
+		Provisioner := fmt.Sprintf("%v", portworx.PortworxCsi)
+
+		//Number of apps to be deployed
+		NumberOfDeployments := 10
```
Comment (lsrinivas-pure) [inline tests/basic/pure_test.go:1051]: scale to 300

File: tests/basic/pure_test.go [hunk 2]
```diff
@@ -1014,3 +1014,78 @@ var _ = Describe("{RestartPXWhileVolCreate}", func() {
...
+			}
+		})
+		stepLog = fmt.Sprintf("Stop portworx,destroy apps and check if the pvc's are deleted gracefully")
+		Step(stepLog, func() {
+			stepLog := fmt.Sprintf("Stop Portworx")
+			Step(stepLog, func() {
+				log.Infof("Stop volume driver [%s] on node: [%s]", Inst().V.String(), selectedNode.Name)
+				StopVolDriverAndWait([]node.Node{selectedNode})
+			})
+			stepLog = fmt.Sprintf("Destroy Application")
```
Comment (lsrinivas-pure) [inline tests/basic/pure_test.go:1073]: add resize pvc step here

File: tests/basic/pure_test.go (hunk 1 above)
Comment (spai-px) [reply tests/basic/pure_test.go]: Done

File: tests/basic/pure_test.go (hunk 2 above)
Comment (spai-px) [reply tests/basic/pure_test.go:1124]: Done

File: tests/basic/pure_test.go [hunk 3]
```diff
@@ -1014,3 +1014,130 @@ var _ = Describe("{RestartPXWhileVolCreate}", func() {
...
+				Step(stepLog,
+					func() {
+						log.InfoD(stepLog)
+						pvcs, err := GetContextPVCs(ctx)
+						log.FailOnError(err, "Failed to get pvc's from context")
+						for _, pvc := range pvcs {
+							log.InfoD("increasing pvc [%s/%s]  size to %d %v", pvc.Namespace, pvc.Name, 40, pvc.UID)
+							resizedVol, err := Inst().S.ResizePVC(ctx, pvc, 40)
+							log.FailOnError(err, "Not able to increase size")
+							if err != nil && !(strings.Contains(err.Error(), "only dynamically provisioned pvc can be resized")) {
```
Comment (lsrinivas-pure) [inline tests/basic/pure_test.go:1099]: remove this check. it is not required here

File: tests/basic/pure_test.go [hunk 4]
```diff
@@ -1014,3 +1014,130 @@ var _ = Describe("{RestartPXWhileVolCreate}", func() {
...
+				stepLog = fmt.Sprintf("increase volume size %s on app %s's volumes: %v",
+					Inst().V.String(), ctx.App.Key, appVolumes)
+				Step(stepLog,
+					func() {
+						log.InfoD(stepLog)
+						pvcs, err := GetContextPVCs(ctx)
+						log.FailOnError(err, "Failed to get pvc's from context")
+						for _, pvc := range pvcs {
+							log.InfoD("increasing pvc [%s/%s]  size to %d %v", pvc.Namespace, pvc.Name, 40, pvc.UID)
+							resizedVol, err := Inst().S.ResizePVC(ctx, pvc, 40)
```
Comment (lsrinivas-pure) [inline tests/basic/pure_test.go:1097]: take current size and increment it

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**